wagtail/images/permissions.py

```diff
@@ -1,9 +1,8 @@
 from wagtail.core.permission_policies.collections import CollectionOwnershipPermissionPolicy
 from wagtail.images import get_image_model
-from wagtail.images.models import Image
 
 permission_policy = CollectionOwnershipPermissionPolicy(
     get_image_model(),
-    auth_model=Image,
+    auth_model=get_image_model(),
```

Comment:
we should only change this if we're absolutely sure about it... If the permission policy has been using the permissions from the default Image model up until now, this could potentially bring about some unwanted permission-related issues on existing projects